scalalib/src/mill/scalalib/JavaModule.scala

```diff
@@ -1126,14 +1130,18 @@ trait JavaModule
   )
 
   @internal
-  def bspJvmBuildTarget: JvmBuildTarget =
+  def bspJvmBuildTarget: Task[JvmBuildTarget] = Task.Anon {
```

Comment:
I think changing this signature might produce trouble, since plugins extending `JavaModule` in `trait`s could break.

Reply:
I'll leave it in place and add a new method